### PR TITLE
Minor fixup for etcd unit files

### PR DESCRIPTION
**Pull request by redbaron**
nothing major, just small nits found while moving to new master

### CODE REVIEW

File: core/controlplane/config/templates/cloud-config-etcd [hunk 1]
```diff
@@ -251,13 +249,12 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants={{.Etcd.SystemdUnitName}}
+        Requires={{.Etcd.SystemdUnitName}}
```
Comment (mumoshu) [inline core/controlplane/config/templates/cloud-config-etcd]: Wouldn't modifying this to `Requires` result in cfn-signal.service to be kept failed permanently when `etcd-member.service` failed at first but then restarted via `Restart=on-failure`(according to /usr/lib64/systemd/system/etcd-member.service` and the restart was successful?

File: core/controlplane/config/templates/cloud-config-etcd [hunk 2]
```diff
@@ -115,10 +114,9 @@ coreos:
         Description=etcd snapshot
 
         [Service]
-        Type=simple
+        Type=oneshot
```
Comment (mumoshu) [inline core/controlplane/config/templates/cloud-config-etcd]: Thanks! This change seems to work.

File: core/controlplane/config/templates/cloud-config-etcd (hunk 1 above)
Comment (mumoshu) [inline core/controlplane/config/templates/cloud-config-etcd]: @redbaron Any chance you could confirm my comment above?

Comment (redbaron) [inline core/controlplane/config/templates/cloud-config-etcd]: it seems that email responses don't show up here :(

It is `oneshot` service therefore it is never restarted, yet it all worked fine

Comment (mumoshu) [inline core/controlplane/config/templates/cloud-config-etcd]: @redbaron Argh, maybe I had forgotten about that multiple times until now!
Thanks for the confirmation 👍 
LGTM for now.

In the future, perhaps we'd better transform all the oneshot services into simple services with all the useful commands in "ExecStartPre"s, while having `ExecStart=/bin/true`?

**Comment by mumoshu** [comment]
Thanks for your contribution as always, @redbaron 🎉

**Pull request merged**